module/submodules/logs.py

```diff
@@ -218,7 +217,7 @@ def get_ui_availability(self, elt, range_start=None, range_end=None):
                 if '_id' in log:
                     del log['_id']
                 records.append(log)
-            logger.debug("[mongo-logs] %d records fetched from database.", records.count())
+            logger.debug("[mongo-logs] %d records fetched from database.", len(records))
```

Comment:
I don't know about this. As I understand it, `records` is a mongo cursor that behaves like an iterator. `len(records)` will force it to be convertir as a list, while `records.count()` use a mongo method to compute the size of the iterator, wich may be way faster.

Reply:
@maethor: in this place, records is a list not a pymongo Cursor ... as such you cannot use the `count` method and you must use `len` ...

Reply:
OK then :)